contrib/README.md

```diff
@@ -19,7 +19,8 @@ const (
 var Sets = func() model.CustomTemplates {
 	templateContents, err := templatesBox.FindString(SetCustomTemplatePath)
 	if err != nil {
-		panic(err)
```

Comment:
I think we want to revert this -- I think the intended behavior is that we panic if we can't find the template in the specified location